wolfSSL/src/chapter02.md

```diff
@@ -472,6 +588,54 @@ Turns on the use of DTLS, or datagram TLS. This isn't widely supported or used.
 
 Turns on wolfSSL’s RSA key generation functionality. See [Keys and Certificates](chapter07.md#keys-and-certificates) for more information.
 
+#### WOLF_PRIVATE_KEY_ID
+
+FIPS v5 and older doesn't support WOLF_PRIVATE_KEY_ID with PK callbacks. Required for client authentification in some cases.
+
+#### WOLFSSL_WOLFSENTRY_HOOKS
+
+Enables wolfSentry hook support.
+
+#### WOLFSSL_CERT_EXT
+
+Cert exit, key and cert generation feature.
+
+#### WOLFSSL_CERT_REQ
+
+Cert requirement, key, and cert generation feature.
+
+#### WOLFSSL_SSLKEYLOGFILE
+
+Warn if secrets logging is enabled. The SHOW_SECRETS and WOLFSSL_SSLKEYLOGFILE options should only be used for debugging and never in a production environment.
+
+#### WOLFSSL_SSLKEYLOGFILE_OUTPUT
+
+It can be used as an Optional Pre-Master-Secret logging for Wireshark.
+
+#### WOLFSSL_HAVE_CERT_SERVICE
+
+Used by autoconf to see if cert service is available.
+
+#### WOLFSSL_HAVE_WOLFSCEP
+
+Enable feature used by autoconf to see if wolfSCEP is available.
+
+#### WOLFSSL_HAVE_MIN
+
+Needed for use with older versions of xc16.
```

Comment:
This macro is for portability of the library to indicate if MIN/MAX are already defined by the platform. It prevents duplicate definitions.